root/release_group/ReleaseGroupIndex.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../context';
+import Annotation from '../static/scripts/common/components/Annotation';
+import WikipediaExtract
+  from '../static/scripts/common/components/WikipediaExtract';
+import CritiqueBrainzLinks from '../components/CritiqueBrainzLinks';
+import CritiqueBrainzReview
+  from '../static/scripts/common/components/CritiqueBrainzReview';
+import PaginatedResults from '../components/PaginatedResults';
+import TaggerIcon from '../static/scripts/common/components/TaggerIcon';
+import loopParity from '../utility/loopParity';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import FormRow from '../components/FormRow';
+import FormSubmit from '../components/FormSubmit';
+import Relationships from '../components/Relationships';
+import ReleaseDates from '../components/ReleaseDates';
+import ReleaseCountries from '../components/ReleaseCountries';
+import ReleaseLabelList from '../components/ReleaseLabelList';
+import ReleaseCatnoList from '../components/ReleaseCatnoList';
+import formatBarcode from '../static/scripts/common/utility/formatBarcode';
+import * as manifest from '../static/manifest';
+import releaseGroupType from '../utility/releaseGroupType';
+
+import ReleaseGroupLayout from './ReleaseGroupLayout';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +eligibleForCleanup: boolean,
+  +mostPopularReview: CritiqueBrainzReviewT,
+  +mostRecentReview: CritiqueBrainzReviewT,
+  +numberOfRevisions: number,
+  +pager: PagerT,
+  +releaseGroup: ReleaseGroupT,
+  +releases: $ReadOnlyArray<$ReadOnlyArray<ReleaseT>>,
+  +wikipediaExtract: WikipediaExtractT,
```

Comment:
One last thing: there isn’t necessary a Wikipedia extract for each release group.
```suggestion
  +wikipediaExtract: WikipediaExtractT | null,
```